administrator/components/com_joomgallery/helpers/adminconfig.php

```diff
@@ -1046,6 +1051,9 @@ public function delete($id, $force = false)
       return false;
     }
 
+    JPluginHelper::importPlugin('content');
+    $this->_mainframe->triggerEvent('onContentAfterDelete', array(_JOOM_OPTION.'.config', $config));
```

Comment:
$this->_mainframe is not declared in this content, this call will throw an error